cpp/include/cuml/ensemble/randomforest.hpp

```diff
@@ -121,6 +122,22 @@ void build_treelite_forest(ModelHandle* model,
 
 std::vector<unsigned char> save_model(ModelHandle model);
 
+ModelHandle tl_mod_handle(ModelHandle* model,
+                          std::vector<unsigned char>& data);
+
+void build_fil_model(const cumlHandle& handle, fil::forest_t* pforest,
```

Comment:
Is building from just two treelite models enough? Won't it be better to build it from N models?